app/scripts/app.js

```diff
@@ -57,11 +57,14 @@ angular
                 controller: 'GenesCtrl',
                 controllerAs: 'genes'
             })
-            .when('/gene/:geneName/variant/:variantName', {
+            .when('/gene/:geneName/alteration/:alterationName', {
                 templateUrl: 'views/gene.html',
                 controller: 'GeneCtrl',
                 controllerAs: 'gene'
             })
+            .when('/gene/:geneName/variant/:alterationName', {
```

Comment:
Why do you still keep variant in path?

Reply:
This is the old URL format. In portal, we are still referring to the /gene/:geneName/variant/:alterationName. I need to redirect that to the updated URL.